app.js

```diff
@@ -20,14 +20,25 @@ Project.prototype.toHtml = function(){
   $newprojectData.removeClass('template');
   $newprojectData.addClass('');
   $newprojectData.find('div.title').html(this.title);
-  $newprojectData.find('a').attr('href', this.link);
-  // $newprojectData.find('a').html('Here is a link!');
-  $newprojectData.find('img').attr('src', this.img);
-  $newprojectData.find('p').html(this.description);
+  $newprojectData.find('p.date').html(this.date);
+  $newprojectData.find('img.project').attr('src', this.img);
+  $newprojectData.find('a.link').attr('href', this.link);
+  $newprojectData.find('a').html('Check out this site');
+  $newprojectData.find('p.description').html(this.description);
+  $newprojectData.find('p.contributors').html(this.contributors);
   return $newprojectData;
 };
 
-allProjects.forEach(function(object){
-  console.log(object);
-  $('.projects').append(object.toHtml());
+allProjects.forEach(function(somethingDifferent){
+  console.log(somethingDifferent);
+  $('.projects').append(somethingDifferent.toHtml());
 });
+
+// allProjects.handleMainNav = function () {
```

Comment:
**TODO FOR LOST POINTS**

Don't push up commented code. I'll be getting stricter on that and docking points.

That being said, I see you were experimenting around and seeing what happens if you didn't wrap all the click functionality inside a function. Your nav bar obviously still works.

However, I encourage you to still keep wrapping it inside the function per class standards anyway and invoke it later. In other words, uncomment your code and just invoke the entire click function as it should be on Line 44.

The reason for that is because it's better to have something human readable to let you know what that function is doing immediately. If you didn't have ``allProjects.handleMainNav`` labeled anywhere and you just looked at Lines 38-41, how long would it take you to figure out what the heck that code is doing?

Probably a lot longer than it would take if it was just wrapped inside a semantically named function, i.e. ``allProjects.handleMainNav``. 

I hope this makes sense, please ask for clarification if you need it. :)